ipfs-cluster-service/main.go

```diff
@@ -342,8 +344,12 @@ func setupLogging(lvl string) {
 
 func setupAllocation(strategy string) (ipfscluster.Informer, ipfscluster.PinAllocator) {
 	switch strategy {
-	case "disk", "reposize":
-		return disk.NewInformer(), ascendalloc.NewAllocator()
+	case "disk", "disk-freespace":
+		informer := disk.NewInformer()
+		return informer, descendalloc.NewAllocator()
```

Comment:
They're meant to be fully decoupled. You might gather metrics for a bunch of  reasons not related to an allocator. Then, there might be metrics which make sense with several allocators.